### ISSUE TITLE
layout space could be improved in default mode

### ISSUE THREAD

**Issue by ntraut** [neuroanatomy/BrainBox#375]
## Thank you for your feedback.

Is it a [x] bug report or a [ ] feature request?

### Current behaviour

* If this is a bug, please explain how to reproduce the problem 
    * Which tools did you use in which sequence?
    none
    * Is the error concerning a text or a volume annotation? 
    no
    * Which label set has been used for the volume annotation (if applicable)?
    not applicable
    * What is the type and resolution of your data? (Can you provide a link to your data?)
    example data
    * Do not include sensitive information.
    example data
    * Upload the browser error message if you are familiar with web developer tools.
 

I don't know if it's a bug or intended but layout repartition is not optimal in default mode. The annotation description takes a lot of space compared to the image, but it was already the case before. The problem is that the tools are now on top of the image, so it feels that too many things are on a restricted space when there is space around. Even after trying to move the tools at the bottom right, it is still above important parts of the image.

<img width="1512" alt="image" src="https://github.com/neuroanatomy/BrainBox/assets/22977927/bddf01ec-605e-449f-8f43-81644babd5fd">

### Expected behaviour

Something like before with the tools not on top of the image would be ideal, if not possible maybe the image could take more space so there is more unused space on the image where we can move the tools.

<img width="1495" alt="image" src="https://github.com/neuroanatomy/BrainBox/assets/22977927/a8ce57cd-f143-4f01-a182-967494fed451">

### Version information (for bug reports)

* **Browser + version**:
chrome latest
* **Your OS + version**:
mac os latest
* Paste the output of these commands:
```
node --version && npm --version
```
not using my local version

**Comment by r03ert0** [comment]
maybe the chat could be closed by default when the window is so small? It could be then open by default in full screen mode

**Comment by r03ert0** [comment]
we could also have a display mode where only the slider and the axial, coronal and sagittal buttons are enabled (but the fix of collapsing the chat by default is easier)

**Comment by r03ert0** [comment]
Yes, don't bother putting the tools at the bottom (it would make it more difficult to embed microdraw in another webpage)

**Comment by ntraut** [comment]
I guess that we need user feedback to know if people are annoyed or not by the new layout. That was one of the main reasons of setting the dev server (https://brainbox.connect-project.io/), but for now we haven't got a lot of feedback from it. Anyway we'll see after the merge...

**Comment by ntraut** [comment]
I updated the dev server to reflect last changes. It will be important to see what users think once the PR will be merged.
